eudemo/eudemo/equilibria/_designer.py

```diff
@@ -131,7 +133,9 @@ def __init__(
     ):
         super().__init__(params, build_config)
         self.file_path = self.build_config.get("file_path", None)
-        self.plot_optimisation = self.build_config.get("plot_optimisation", False)
+        self.diagnostic_plotting = self.build_config.get(
```

Comment:
build config comes from ~user json (at the moment at least) it is not possible to initialise the options directly from the json it has to be hard coded or the build config modified. 
An easyish fix here would be to assume it is a dict at this point and convert here. In future hopefully this will be possible to set up the options from read in when we start to get pydantic going. For now maybe:
```python
 self.diagnostic_plotting = PicardDiagnosticOptions(*self.build_config.get( "diagnostic_plotting", {}))